src/services/adresregister.api-service.ts

```diff
@@ -0,0 +1,151 @@
+import { inject } from 'aurelia-framework';
+import { HttpClient } from 'aurelia-http-client';
+import { RestMessage } from '../utilities/message/restMessage';
+import { MessageParser } from '../utilities/message/messageParser';
+import { sortBy } from 'lodash';
+import { IAdresregisterAdres, IGemeente, IGewest, ILand, IPostinfo, IProvincie, IStraat } from './models/locatie';
+import { Niscode } from './models/niscode.enum';
+
+declare const oeAppConfig: any;
+
+@inject(HttpClient)
+export class AdresregisterService {
+  private landen: ILand[] = [];
+  private provincies: IProvincie[] = [];
+  private gemeenten: IGemeente[] = [];
+
+  private gemeentenVlaamsGewest: IGemeente[] = [];
+  private gemeentenWaalsGewest: IGemeente[] = [];
+  private gemeentenBHGewest: IGemeente[] = [];
+
+  constructor(
+    private http: HttpClient
+  ) {
+    this.http.configure(x => {
+      x.withBaseUrl(oeAppConfig.crabpyUrl);
+      x.withHeader('Accept', 'application/json');
+      x.withHeader('X-Requested-With', '');
+      x.withInterceptor({
+        responseError(res) {
+          RestMessage.display({ result: MessageParser.parseHttpResponseMessage(res) });
+          return res;
```

Comment:
`throw res` zoals in de andere nieuwe services? nog foutafhandeling toevoegen waar nodig. Properste vind ik in het adres-crab component voor de meeste gevallen => 
```js
try {
   adresregisterService.function()
   ...
catch(e){
   log.error(e)/foutafhandeling 
}
```